### PR TITLE
add tasks to files 1.js 2.js 3.js

### PR DESCRIPTION
**Pull request by alexanderxp**

### CODE REVIEW

File: 1.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+1. Написать функцию, которая принимает массив числовых значений, 
+и возвращает минимальное и максимальное значение массива
+*/
+
+"use strict"
+
+let arr = [10, 2, 3, 15, 46, 98];
+
+function maxNum(arr){
+ let max = arr[0];
```
Comment (StanislawVictorovich) [inline 1.js]: please add data type filtering to avoid crashing web app

File: 2.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+и возвращает среднее значение
+*/
+
+"use strict"
+
+let arr = [10, 2, 3, 15, 46, 98];
+// let arr = [1, 2, 3, 4, 5, 6, 7];
+
+function averageNum(array) {
+    return array.reduce((a, b) => (a + b)) / array.length;
```
Comment (StanislawVictorovich) [inline 2.js]: Don't use Array.Prototype methods

File: 3.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+3. Написать функцию, которая принимает массив числовых значений 
+и проверяет является ли массив отсортирован (возвращает true/false)
+*/
+
+"use strict"
+
+let arr = [1, 2, 3, 4, 5, 6, 7];
+//  let arr = [7, 6, 8, 138, 11];
+
+let isArrSorted = arr => arr.every((value,item,array) => !item || array[item-1] <= value);
```
Comment (StanislawVictorovich) [inline 3.js]: Don't use Array.Prototype methods

File: 1.js (hunk 1 above)
Comment (alexanderxp) [reply 1.js]: Done

File: 2.js (hunk 2 above)
Comment (alexanderxp) [reply 2.js]: Done.

File: 3.js (hunk 3 above)
Comment (alexanderxp) [reply 3.js]: Done.

File: 1.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+1. Написать функцию, которая принимает массив числовых значений, 
+и возвращает минимальное и максимальное значение массива
+*/
+
+"use strict"
+
+let arr = [10, 2, 3, 15, 46, 98];
+
+function maxNum(array){
+  if(!(array instanceof Array)){
```
Comment (StanislawVictorovich) [inline 1.js:13]: Separate statements with the line breaks

File: 3.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+//  let arr = [7, 6, 8, 138, 11];
+
+function isArrSorted (array){
+  if(!(array instanceof Array)){
+    return null;
+  };
+  let num;
+  for(num = 0; num > array.length - 1; num ++);{
+    if (array[num] < array[num+1]) {
+      return true;
```
Comment (StanislawVictorovich) [inline 3.js]: Rewrite if else statement to remove else

File: 1.js (hunk 4 above)
Comment (alexanderxp) [reply 1.js:13]: Done.

File: 3.js (hunk 5 above)
Comment (alexanderxp) [reply 3.js]: Done.

**Pull request merged**